rule/max_public_structs.go

```diff
@@ -56,7 +57,7 @@ func (r *MaxPublicStructsRule) Apply(file *lint.File, arguments lint.Arguments)
 
 	if walker.current > r.max {
 		walker.onFailure(lint.Failure{
-			Failure:    "you have exceeded the maximum number of public struct declarations",
+			Failure:    fmt.Sprintf("you have exceeded the maximum number (%d) of public struct declarations", r.max),
```

Comment:
Also, what happen if max < 0, like -1

Number of struct will always be positive, do bigger than -1

Also, what is the point of reporting an error about something bigger than -1 in the error message

Your changes are not exactly about that, but I thought about it by reading the code